config.yaml

```diff
@@ -40,3 +40,31 @@ options:
       to go into error state until the change is reverted.
     default: ""
     type: string
+  containerd_env:
```

Comment:
I'd strongly recommend a different approach here - allowing big strings like this is a highway to issues in production.

Instead I'd consider adding individual config items for the things in here that you want to enable config for, for example `containerd_http_proxy` or `containerd_ulimit` or similar.